wiz-kubernetes-connector/templates/service-account-create-connector.yaml

```
@@ -0,0 +1,55 @@
+{{- if .Values.autoCreateConnector.enabled }}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ .Values.autoCreateConnector.serviceAccount.name }}
+  namespace: {{ .Release.Namespace | quote }}
+  labels:
+    {{- include "wiz-kubernetes-connector.labels" . | nindent 4 }}
+  annotations:
+    "helm.sh/hook": pre-install,pre-upgrade
+    "helm.sh/hook-delete-policy": before-hook-creation,hook-succeeded
+  {{- with .Values.autoCreateConnector.serviceAccount.annotations }}
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: {{ .Values.autoCreateConnector.serviceAccount.name }}
+  labels:
+    {{- include "wiz-kubernetes-connector.labels" . | nindent 4 }}
+  annotations:
+    "helm.sh/hook": pre-install,pre-upgrade
+    "helm.sh/hook-delete-policy": before-hook-creation,hook-succeeded
+rules:
+  - apiGroups: ["*"]
+    resources: ["secrets"]
+    resourceNames: [{{ include "wiz-kubernetes-connector.connectorSecretName" . | trim | quote }}]
+    verbs: ["update"]
+  - apiGroups: ["*"]
```

Use the core api group instead (empty string)

```suggestion
  - apiGroups: [""]
    resources: ["secrets"]
    resourceNames: [{{ include "wiz-kubernetes-connector.connectorSecretName" . | trim | quote }}]
    verbs: ["update"]
  - apiGroups: [""]
```